### PR TITLE
Remove Show button for hotspot metric configurations

### LINKED ISSUE #310: Hotspot Metric configuration showing is broken
It fails with "Invalid Reading Group", even though Hotspot Metric Configurations shouldn't care about reading groups. It may be even better to remove the button completely, as there is no relevant information to show.

### PR DESCRIPTION
**Pull request by danielkza**
This removes the `Show` button for hotspot metric configurations on the
configurations page.

Signed-off-by: Daniel Miranda <danielkza2@gmail.com>

Fixes #310.

### CODE REVIEW

File: app/views/kalibro_configurations/_metric_configurations.html.erb [hunk 1]
```diff
@@ -1,22 +1,22 @@
 <tr>
   <td><%= metric_configuration.metric.name %></td>
   <td><%= metric_configuration.metric.code %></td>
-  <% unless metric_configuration.metric.is_a? KalibroClient::Entities::Miscellaneous::HotspotMetric %>
+  <% unless hotspot_metric_configuration?(metric_configuration) %>
```
Comment (rafamanzo) [inline app/views/kalibro_configurations/_metric_configurations.html.erb]: Nice

**Comment by rafamanzo** [comment]
CHANGELOG?

**Comment by diegoamc** [comment]
:+1:

**Comment by danielkza** [comment]
@rafamanzo Fixed, thanks for the reminder.

**Comment by rafamanzo** [comment]
Thanks

**Pull request merged**